serverless/puppeteer-screenshots/src/configs/config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+const launchOptionForLambda = [
+    // error when launch(); No usable sandbox! Update your kernel
+    '--no-sandbox',
+    // error when launch(); Failed to load libosmesa.so
+    '--disable-gpu',
+    // freeze when newPage()
+    '--single-process',
+];
+
+const localChromePath = path.join('headless_shell.tar.gz');
+const remoteChromeS3Bucket = process.env.CHROME_BUCKET;
+const remoteChromeS3Key = process.env.CHROME_KEY || 'headless_shell.tar.gz';
```

Comment:
If we make this function stay warm, we can download the built chrome executable and avoid expensive start up times, as well as avoid expensive deploy/build times. Then we can just update chrome when our screenshots start looking weird after a very long time, when there are potential incompatibilities with new front end code and an old version of chrome we haven't updated in a long time.